book/src/guides/coding-conventions.md

```diff
@@ -0,0 +1,64 @@
+# Coding Conventions
```

Comment:
We have these  hosted on docs.sui.io currently. Do you want to move them here? what's the plan?

Reply:
I want to split what we have there. But probably make the book the main place. It’s tricky to find conventions in the current docs website.